dev/update-examples.sh

```diff
@@ -16,20 +16,49 @@ echo "---" >> $INDEX
 echo "maxdepth: 1" >> $INDEX
 echo "---" >> $INDEX
 
-rm -f "examples/doc/source/*.md"
+rm -f examples/doc/source/*.md
 
 cd examples/
 for d in $(printf '%s\n' */ | sort -V); do
   example=${d%/}
-  # For each example, copy the README into the source of the Example docs
-  [[ $example != doc ]] && cp $example/README.md $ROOT/examples/doc/source/$example.md 2>&1 >/dev/null
-  # For each example, copy all images of the _static folder into the examples
-  # docs static folder
-  [[ $example != doc ]] && [ -d "$example/_static" ] && {
-    cp $example/_static/**.{jpg,png,jpeg} $ROOT/examples/doc/source/_static/ 2>/dev/null || true
-  }
-  # For each example, insert the name of the example into the index file
-  [[ $example != doc ]] && (echo $INSERT_LINE; echo a; echo $example; echo .; echo wq) | ed $INDEX 2>&1 >/dev/null
+
+  if [[ $example != doc ]]; then
+
+    # For each example, copy the README into the source of the Example docs
+    cp $example/README.md $ROOT/examples/doc/source/$example.md 2>&1 >/dev/null
+    cp $example/README.md $ROOT/examples/doc/source/$example.md 2>&1 >/dev/null
```

Comment:
@charlesbvll, why copy twice the same file?

Reply:
@tanertopal Those lines were supposed to be removed, it should be fixed now